.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ parameters:
   OLM_VERSION:
     type: "string"
     default: "v0.21.2"
+  CHARTMUSEUM_VERSION:
+    type: "string"
+    default: "3.9.0"
```

Comment:
I'm also updating from `2.14.2` to `3.9.0` here, but I'll roll the changes back if the PR fails.

Reply:
It seems that it worked 😄